pkg/output/lumberjack/lumberjack.go

```diff
@@ -0,0 +1,155 @@
+// Licensed to Elasticsearch B.V. under one or more agreements.
+// Elasticsearch B.V. licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+package tcp
```

Comment:
should this be `package lumberjack`??

Reply:
Yes. I'm surprised I got as far as I did without running into an error. 😆